Release 2.2.0 (#3103)
timescale/timescaledb: CHANGELOG.md

## Unreleased

## 2.2.0 (2021-04-13)

This release adds major new features since the 2.1.1 release.
We deem it moderate priority for upgrading.

This release adds the Skip Scan optimization, which significantly 

> improves the performance of queries with DISTINCT ON. This 
> optimization is not available for queries on distributed hypertables.

Reviewer: ```suggestion
improves the select performance of queries on hypertables with
DISTINCT ON. This query optimization is not yet available for
distributed hypertables.
```

Reviewer: Does skip scan work on relational tables as well, or just hypertables?

Author: It seems to me that it works on plain PG tables too. @svenklemm Am I correct?

Reviewer 2: works on both

> optimization is not available for queries on distributed hypertables.

Reviewer 3: should we say "_currently_ not available ... on distributed hypertables"?

## Unreleased

## 2.2.0 (2021-04-13)

This release adds major new features since the 2.1.1 release.
We deem it moderate priority for upgrading.

> This release adds the skip-scan optimization, which significantly 

Reviewer 2: should be `Skip Scan`

Author: I don't see that `Skip Scan` is used anywere. Either `SkipScan` or the [actual commit message](https://github.com/timescale/timescaledb/pull/3000) contains:
>This patch implements a skip-scan;

Reviewer 2: SkipScan is the node name which follows camelcase convention for node names but for external messaging we decided to use "Skip Scan"

> improves performance of queries with DISTINCT ON. This optimization is 

Reviewer 3: Not 100% sure, but this reads better to me

```suggestion
improves the performance of queries with DISTINCT ON. This optimization is 
```

not available for queries on distributed hypertables.

For multinode, this release adds a function to create a distributed 

> restore point, which allows the consistent restore of a multinode 

Reviewer 3: ```suggestion
restore point, which allows performing a consistent restore of a multi-node 
```

cluster from a backup. The release also includes improvements to 

> distributed query performance and memory usage.

Reviewer 3: Not a biggie, but the performance and memory usage stuff is immediately repeated below in more detail.
